### ISSUE TITLE
Do not show the validation error message in pipe syntax

### ISSUE THREAD

**Issue by Daniyal-Javani** [BenSampo/laravel-enum#133]
The following code works fine:
`'status' => ['required', new EnumKey(OrderStatus::class)]`
and returns this
`"The key you have entered is invalid."`

But when we use the pipe syntax like this:
`'status' => 'required|enum_key:' . OrderStatus::class`
it returns:
`"validation.enum_key"`

If it has intended to fixed, here are my suggestions
The easiest one:
Copy and paste the string like this
```
Validator::extend('enum_key', function ($attribute, $value, $parameters, $validator) {
    $enum = $parameters[0] ?? null;

    return (new EnumKey($enum))->passes($attribute, $value);
}, 'The key you have entered is invalid.');
```

Or do some messy things like
```
Validator::extend('enum_key', function ($attribute, $value, $parameters, $validator) {
    $enum = $parameters[0] ?? null;

    return (new EnumKey($enum))->passes($attribute, $value);
}, ((new \ReflectionClass(EnumKey::class))->newInstanceWithoutConstructor())->message());
```

Please let me know your idea about the solution of the issue.

**Comment by BenSampo** [comment]
We should probably be using translation files for both.

See 
https://laravel.com/docs/master/packages#translations
and
https://laracasts.com/discuss/channels/laravel/package-development-custom-validation-messages?page=1

**Comment by BenSampo** [comment]
Resolved by #134
